README.md

```diff
@@ -1,125 +1,183 @@
-  
-Additionally there's a compatibility checker, that prints a list of incompatible or outdated addons in the server console. More on this in our documentation  [here](https://docs.ttt2.neoxult.de/troubleshooting/#addon-checker).  
-**Hint:** You do not need addons like “Better Equipment Menu”, “Drowning Indicator” or “TTT Sprint”, because features like these are included in TTT2 by default.  
-  
+We know that TTT lives from its huge community and all its great addons.
+Because of this, **compatibility** is one of the highest priorities.
+Almost every item that works with TTT also works with TTT2.
+HUD and role related addons are the exceptions.
+
+Additionally we have introduced a compatibility checker.
+On server start it prints incompatible or outdated addons to the server console.
+More on this in our documentation [here](https://docs.ttt2.neoxult.de/troubleshooting/#addon-checker).
+
+TTT2 also features some additions which TTT needed addons for:
+
+* overhauled equipment menu
+* drowning indicator
+* sprinting and stamina
+
 ## NEW GAMEMODES
 
-There are a few new gamemodes based on TTT2. Click on these icons to open a list of needed addons to play these currently available official gamemodes.  
+There are a few new gamemodes based on TTT2.
+Click on these icons to open the steam collection to these official gamemodes.
 
-[![https://i.imgur.com/5JAsxin.png](https://i.imgur.com/5JAsxin.png)](https://steamcommunity.com/sharedfiles/filedetails/?id=1672031318)
-[![https://i.imgur.com/qwEkCPb.png](https://i.imgur.com/qwEkCPb.png)](https://steamcommunity.com/sharedfiles/filedetails/?id=1672014264)
-[![https://i.imgur.com/WVuPmxP.png](https://i.imgur.com/WVuPmxP.png)](https://steamcommunity.com/sharedfiles/filedetails/?id=1737047642)
+[![TTT2 Totem](https://i.imgur.com/5JAsxin.png)](https://steamcommunity.com/sharedfiles/filedetails/?id=1672031318)
+[![TTT2 Fate](https://i.imgur.com/qwEkCPb.png)](https://steamcommunity.com/sharedfiles/filedetails/?id=1672014264)
+[![TTT2 Heroes](https://i.imgur.com/WVuPmxP.png)](https://steamcommunity.com/sharedfiles/filedetails/?id=1737047642)
 
 ## SETUP
 
-The setup is fairly easy. Just subscribe to TTT2 and the addons you want to use and get started. It’s recommended to use ULX to set it up ingame, there’s an [ULX addon for TTT2](https://steamcommunity.com/sharedfiles/filedetails/?id=1362430347). Commands for the admin shop editor etc. are found in [this wiki article](https://docs.ttt2.neoxult.de/).
-  
-<br />
-<br />
+The setup is fairly easy. Just subscribe to TTT2 and the addons you want to use.
+
+It’s recommended to use ULX to set it up ingame, there’s an [ULX addon for TTT2](https://steamcommunity.com/sharedfiles/filedetails/?id=1362430347).
+Commands for the admin shop editor etc. are found in [this wiki article](https://docs.ttt2.neoxult.de/).
 
 <p align="center">
-	<img src="https://i.imgur.com/pbZHURE.png">
+    <img src="https://i.imgur.com/pbZHURE.png" alt="'Custom Role Support' Banner">
```

Comment:
is this change from tab to spaces intentional?

Reply:
not really. i wanted to scrap all html tags from the markdown and played around with different things, but there's no way to center images without html.
Seems like this happened while changing that back (i have no formatter for markdown installed, hence nothing that could respect the .editorconfig)